docs/tutorials/projects/filtered-camera-module.md

```diff
@@ -0,0 +1,400 @@
+---
+title: "Selectively capture data using filtered-camera"
+linkTitle: "Filtered Camera"
+weight: 6
+type: "docs"
+description: "Use the filtered-camera module to selectively capture images."
+images: ["/tutorials/filtered-camera-module/viam-figure-preview.png"]
+imageAlt: "The wooden Viam figure being detected by a transform camera"
+tags: ["camera", "vision", "detector", "mlmodel", "data"]
+no_list: true
+viamresources: ["camera", "vision", "mlmodel", "data"]
+level: "Intermediate"
+date: "2023-12-20"
+# updated: ""
+cost: "0"
+---
+
+{{< imgproc src="/tutorials/filtered-camera-module/viam-figure-preview.png" alt="The promotional Viam wooden figure we give out at events, being correctly detected with a 0.97 confidence threshold" resize="400x"  class="alignright" >}}
+
+With the data management service, a Viam machine can capture data from a variety of components and sync that data to the Viam app.
+However, if your machine captures a large volume of data, especially image data such as pictures, you may wish to control which specific images are captured or uploaded.
+
+For example, imagine that you have positioned your machine's camera to view a busy city street.
+Your machine will happily capture as many pictures as its configured capture rate dictates, resulting in potentially a large number of images captured over the course of a day.
+If you wanted to see if any fancy sports cars might have driven by, you might have to search through many images to find out!
+
+Instead, you can use the `filtered-camera` module to be able to selectively capture and sync only those images that meet the specific criteria you've outlined in a machine learning (ML) model.
+For example, you could train an ML model that is focused on sports cars, and only capture images from the camera feed when a sports car is detected in the frame.
+
+This tutorial will demonstrate using the `filtered-camera` module to selectively capture images only when a specific object is detected within the camera feed: the Viam wooden figure.
```

Comment:
```suggestion
In this tutorial, you will learn how to use the `filtered-camera` module to selectively capture images only when a specific object is detected within the camera feed: the Viam wooden figure.
```

Reply:
Yesss I will use this phrasing forever now, I never know how to say this without passive future second person singular tense case voice!